### PR TITLE
:sparkles: Add available notifications

### PR DESCRIPTION
**Pull request by kevinscholz**
@jonas612 could you especially have an eye on whether this is understandable and whether the mitigations are clear? 

Depends on https://github.com/setopsco/docs/pull/30 and https://github.com/setopsco/aws-provider/pull/275.

### CODE REVIEW

**Comment by kevinscholz** [comment]
Will succeed when https://github.com/setopsco/docs/pull/30 is merged

File: content/latest/user/configuration/notifications.md [hunk 1]
```diff
@@ -4,9 +4,39 @@ title: Notifications
...
 
-Notifications have a `type` and a `target`. The Type specifies the integration to use and the Target represents the recipient called in case of an alert. See below for a list of types supported.
+## Available Notifications and mitigation
+
+SetOps currently notifies about the following issues with your application:
+
+| Affected entity | Name | Description | Mitigation |
+|------|-----------------|-------------|------------|
+| App | Replicas Not Scaled | The App and the corresponding containers could not be started and scaled as configured with *Scale* in the last 10 minutes. | Most likely the containers cannot be started because of an error within the application itself. Check the logs to find the these kind of errors. If there are no errors in the logs, there can be two issues: failing health checks or not enough configured memory. First ensure that the health checks indeed work as configured and then try to increase the memory for your App. |
```
Comment (jonas612) [inline content/latest/user/configuration/notifications.md:15]: The table looks a bit cluncy. Do you have ideas for a better formatting? I would still accept this 😉 

image

**Review by jonas612 — APPROVED** [review]
@kevinscholz  very in-depth and conclusive description of issues and mitigation 👍 

The format is not optimal (see comment). However, I would accept this for now 🚀

**Review by jonas612 — DISMISSED** [review]
Found two minor problems:
- a missing `|` separator in the table 
- a broken link to `services.md`

Should all work fine now! 🚀

**Comment by kevinscholz** [comment]
@jonas612 I changed the format. Do you like this one better?

**Pull request merged**